.github/workflows/golicense.yml

```diff
@@ -35,7 +35,7 @@ jobs:
     steps:
     - uses: actions/checkout@v3
     - name: Set up Go using version from go.mod
-      uses: actions/setup-go@v3
+      uses: actions/setup-go@v4
```

Comment:
you probably want to set `cache-dependency-path` here as well, to be safe
this time you would set it to `'**/go.sum'`